csharp/Platform.Setters/Setter[TResult].cs

```diff
@@ -5,32 +5,26 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>
-    /// Represents the setter.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the setter.</para>
+    /// <para>Представляет установщик.</para>
     /// </summary>
     /// <seealso cref="Setter{TResult, bool}"/>
     public class Setter<TResult> : Setter<TResult, bool>
     {
         /// <summary>
-        /// <para>
-        /// Initializes a new <see cref="Setter"/> instance.
-        /// </para>
-        /// <para></para>
+        /// <para>Initializes a new <see cref="Setter"/> instance.</para>
```

Comment:
Above you have written `Initializes a new instance of the class` 
Stick to the same style. Choose between this or that variant and check how microsoft does